cmake/FindFreeRTOS.cmake

```diff
@@ -7,6 +7,10 @@ list(REMOVE_DUPLICATES FreeRTOS_FIND_COMPONENTS)
 
 set(FreeRTOS_HEAPS 1 2 3 4 5)
 
+if(NOT FREERTOS_PATH)
+    set(FREERTOS_PATH $ENV{FREERTOS_PATH} CACHE PATH "Path to FreeRTOS")
+endif()
+
```

Comment:
This one coding scheme is inconsistant with the other 3.
With quick glance I do not expect any difference in behavior and I find this one clearer (personally).
What do you think to change the other 3 implementations to fit syntax of this one ?

Reply:
Sure.